pubtools/_quay/tag_images.py

```diff
@@ -309,6 +311,8 @@ def tag_images(
 
     with executor_class() as executor:
         executor.skopeo_login(quay_user, quay_password)
+        if source_quay_user and source_quay_password:
```

Comment:
There's no source_quay_host (I need to extract it from source_ref). But if user won't provide source_quay_user and password  (which won't be provided unless explicitly needed) it won't login. I think it should be enough to avoid unnecessary login

Reply:
source_quay_user and source_quay_password are required target settings, so they are always provided by pub

Reply:
fixed